index.js

```diff
@@ -1,18 +1,38 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector('#header')
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
-
+header.style.color = "aqua"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+console.log('This is what the traveler object looks like: ', traveler)
+const profileDiv = document.querySelector('#profile')
+profileDiv.innerHTML = `
+  <img src=${traveler.photo} alt=${traveler.name}>
+  <h2>${traveler.name}</h2>
+  <p>AKA: <em>${traveler.nickname}</em></p>
+`;
 
```

Comment:
Good good! While this works just fine, for updating elements that _already exist_ on the DOM, consider first finding the element/s you're wanting to update and then update it/them instead of overwriting it/them with innerHTML.
```js
const profileImg = document.querySelector("#profile img") // find existing element
profileImg.src = traveler.photo // update src attribute
profileImg.alt = traveler.name // update alt attribute
const profileH2 = document.querySelector("#profile h2") // find existing element
profileH2.textContent = traveler.name // update text displayed by this element
const profileEm = document.querySelector("#profile em") // find existing element
profileEm.textContent = traveler.nickname // update text displayed by this element
const likes = document.querySelector("#profile .likes") // find existing element
likes.textContent = `${traveler.likes} Likes` // update text displayed by this element
```